src/modules/voxelformat/GLTFFormat.cpp

```diff
@@ -0,0 +1,426 @@
+/**
+ * @file
+ */
+
+#include "GLTFFormat.h"
+#include "app/App.h"
+#include "core/Color.h"
+#include "core/Log.h"
+#include "core/Var.h"
+#include "core/concurrent/Lock.h"
+#include "core/concurrent/ThreadPool.h"
+#include "voxel/CubicSurfaceExtractor.h"
+#include "voxel/IsQuadNeeded.h"
+#include "voxel/MaterialColor.h"
+#include "voxel/Mesh.h"
+#include "voxel/VoxelVertex.h"
+#include <SDL_timer.h>
+#include <strstream>
```

Comment:
is there is other way in tinygltf than iostreams? I would rather not include iostreams as the blow up the binary size by a huge amount on some systems.

Reply:
https://github.com/mgerhardy/vengi/commit/59e69f62ed2da6799cef4a1a2fdc52b028e274b3